task_runner

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# ----------------------------------------------------------------------------
+# Copyright (c) 2013--, scikit-bio development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+TARGET=skbio
+
+if [ ${2} ]; then
+  TARGET=${2}
+fi
+
+COMMAND=${1}
+
+case "${COMMAND}" in 
+  "tests" )
+    set -x
+    nosetests ${TARGET} --with-doctest --ignore DO_NOT_IGNORE_ANYTHING
+    pep8 ${TARGET}
```

Comment:
Can you update the `pep8` and `flake8` commands to also check `setup.py` and `checklist.py`? This check is performed in .travis.yml.